docs/_build/html/_sources/quickstart.rst.txt

```diff
@@ -0,0 +1,180 @@
+    b_tt = ttax.random_.tensor(rng, [3, 4, 5], tt_rank=30, dtype=dtype)
+    sum_tt = a_tt + b_tt
+    prod_tt = a_tt * b_tt
+    twice_a_tt = 2 * a_tt
+    
+.. important::
+    Most operations on TT-tensors increase the TT-rank. After applying a sequence of operations the TT-rank can increase by too much and we may want to reduce it. To do that there is a rounding operation, which finds the tensor that is of a smaller rank but is as close to the original one as possible.
+    
+.. code-block:: python
+
+    rounded_prod_tt = ttax.round(prod_tt)
+    
+    a_max_tt_rank = np.max(a_tt.tt_ranks)
+    b_max_tt_rank = np.max(b_tt.tt_ranks)
+    exact_prod_max_tt_rank = np.max(prod_tt.tt_ranks)
+    rounded_prod_max_tt_rank = np.max(rounded_prod_tt.tt_ranks)
+    print('The TT-ranks of a and b are %d and %d. The TT-rank '
+          'of their elementwise product is %d. The TT-rank of '
+          'their product after rounding is %d.' % (a_max_tt_rank, 
+          b_max_tt_rank, exact_prod_max_tt_rank, 
+          rounded_prod_max_tt_rank))
+    
+Check that rounded TT-tensor of product converted to the full format is close to the product of full tensors a and b:
+
+.. code-block:: python
+
+    actual_prod = ttax.full(a_tt) * ttax.full(b_tt)
+    prod_full = ttax.full(prod_tt)
+    rounded_prod_full = ttax.full(rounded_prod_tt)
+    np.testing.assert_allclose(actual_prod, rounded_prod_full, 1e-3)
+    np.testing.assert_allclose(actual_prod, prod_full, 1e-3)
+    np.testing.assert_allclose(prod_full, rounded_prod_full, 1e-3)
+    
+Working with TT-matrices
+------------------------
+
+Recall that for 2-dimensional tensors the TT-format coincides with the matrix low-rank format. However, sometimes matrices can have full matrix rank, but some tensor structure (for example a kronecker product of matrices). In this case there is a special object called Matrix TT-format. You can think of it as a sum of kronecker products (although it’s a bit more complicated than that).
+
+Let’s say that you have a matrix of size 8 x 27. You can convert it into the matrix TT-format of tensor shape (2, 2, 2) x (3, 3, 3) (in which case the matrix will be represented with 3 TT-cores) or, for example, into the matrix TT-format of tensor shape (4, 2) x (3, 9) (in which case the matrix will be represented with 2 TT-cores).
+
+.. code-block:: python
+
+    rng = jax.random.PRNGKey(41)
+    a_matrix_tt = ttax.random_.matrix(rng, ((2, 2, 2), (3, 3, 3)), tt_rank=4, dtype=dtype)
+    twice_a_matrix_tt = 2.0 * a_matrix_tt
+    prod_tt = a_matrix_tt * a_matrix_tt
+
+.. code-block:: python
+
+    rng1, rng2 = jax.random.split(jax.random.PRNGKey(0))
+    dtype = jnp.float32
+    left_shape = (2, 3, 4)
+    sum_shape = (4, 3, 5)
+    right_shape = (4, 4, 4)
+    tt_a = ttax.random_.matrix(rng1, (left_shape, sum_shape), tt_rank=3, dtype=dtype)
+    tt_b = ttax.random_.matrix(rng2, (sum_shape, right_shape), tt_rank=[1, 4, 3, 1],
```

Comment:
ttax.random.matrix instead of ttax.random_.matrix (everywehere in the user facing code -- we need to use random_ only inside the code of the library itself)